src/components/SingleRev.jsx

```diff
@@ -1,34 +1,68 @@
 import { useEffect, useState } from "react";
-import { getReviewsList } from "../api";
+import { getComments, getReviewsList } from "../api";
 import { useParams } from "react-router-dom";
 
+export const SingleRev = ({ loading, setLoading }) => {
+  const { reviewId } = useParams();
+  const [oneRev, setOneRev] = useState({});
+  const [comments, setComments] = useState([]);
 
-
-export const SingleRev = ({loading, setLoading}) => {
-  const {reviewId} = useParams()
-  const [oneRev, setOneRev] = useState({})
   useEffect(() => {
+    setLoading(true);
     getReviewsList(reviewId).then((review) => {
       setOneRev((rev) => {
         setLoading(false);
-        console.log(review);
         return review;
       });
     });
-    
   }, [reviewId]);
-  return (
+
+  useEffect(() => {
+    setLoading(true);
+    getComments(reviewId).then((comment) => {
+      setComments((comm) => {
+        setLoading(false);
+        return comment;
+      });
+    });
+  }, [reviewId]);
+  const handleVote = () => {};
+
+  const handleVoteComm = () => {};
+  return loading ? (
+    <p>Loading, Please wait ...</p>
+  ) : (
     <section>
       <ul>
-      <li key={oneRev.review_id}>
-<p>Title: {oneRev.title}</p> <br></br>
-<img src={oneRev.review_img_url} alt ="" width="500" height="400"/><br></br>
-<p>Category: {oneRev.category}</p>
-<p>Owner: {oneRev.owner}</p> 
-<p>Designer: {oneRev.designer}</p>
-<p>Posted at: {oneRev.created_at}</p>
-<p>Comments: {oneRev.comment_count}</p></li>
+        <li key={oneRev.review_id}>
+          <p>Title: {oneRev.title}</p> <br></br>
+          <img src={oneRev.review_img_url} alt="" width="500" height="400" />
+          <br></br>
+          <p>Category: {oneRev.category}</p>
+          <p>Owner: {oneRev.owner}</p>
+          <p>Category: {oneRev.category}</p>
+          <p>Designer: {oneRev.designer}</p>
+          <p>Posted at: {oneRev.created_at}</p>
+          <button onClick={handleVote}> 👍 {oneRev.votes}</button>
+        </li>{" "}
+        <br></br>
+        <h4>Comments:</h4>
+        <ul>
+          {comments.map((com) => {
```

Comment:
It would be good to have some visual feedback if there are no comments, so that the user knows it's because there aren't any rather than a failure with them loading in.